### PR TITLE
add review ownership for apple and configurability

### PR DESCRIPTION
**Pull request by aiuto**
cc: @keith

### CODE REVIEW

**Review by keith — APPROVED** [review]

**Review by brentleyjones — APPROVED** [review]

File: CODEOWNERS [hunk 1]
```diff
@@ -66,6 +69,14 @@ android/ @ahumesky
 /tools/test/collect_cc_coverage.sh @lberki
 /tools/test/CoverageOutputGenerator/ @lberki
 
+# Configurability
+
+/src/main/java/com/google/devtools/build/lib/analysis @bazelbuild/configurability
+/src/main/java/com/google/devtools/build/lib/rules/platform @bazelbuild/configurability
+/src/test/java/com/google/devtools/build/lib/analysis @bazelbuild/configurability
+/src/test/java/com/google/devtools/build/lib/rules/platform @bazelbuild/configurability
+/tools/platforms @bazelbuild/configurability @bazelbuild/configurability
```
Comment (gregestren) [inline CODEOWNERS:78]: Repeat?

Comment (aiuto) [reply CODEOWNERS:78]: It only look that way  the paths switch  /main/ => /test/.
It's the broken java convention of not having the tests in the same directory as the code.

Comment (brentleyjones) [reply CODEOWNERS:78]: I think he means the double team mention on this line.

Comment (aiuto) [reply CODEOWNERS:78]: Ah, yes. It's hard to  see your own typos.

Comment (aiuto) [reply CODEOWNERS:78]: Fixed in the merge

**Review by gregestren — APPROVED** [review]

**Comment by gregestren** [comment]
Is this ready to merge?

Also, what's the policy on which Googler merges changes approved by a non-Googler?

**Comment by aiuto** [comment]
> Is this ready to merge?
Yes. The CL is in your review list.

> Also, what's the policy on which Googler merges changes approved by a non-Googler?
I don't think there is a distinct one, since a Googler still has to approve the true merge.
In this case, the non-Google reviewers are appropriate because the change adds work to their plate.

**Comment by gregestren** [comment]
> > Is this ready to merge?
> > Yes. The CL is in your review list.

You're right - somehow that slipped by me. All the delay's on me. Apologies.

> > Also, what's the policy on which Googler merges changes approved by a non-Googler?
> > I don't think there is a distinct one, since a Googler still has to approve the true merge.
> > In this case, the non-Google reviewers are appropriate because the change adds work to their plate.

The Googler's responsibility is simply to get it merged, right? And trust that the non-Googler's approval means sufficient review was done?

I want to clarify that merge sequence so that doesn't become the same productivity blocker the ownership change is trying to alleviate.

I believe current policy is to route Apple PRs touching core Bazel to `team-rules-CPP` and assign to its [team lead](https://bazel.build/maintaining/maintainers-guide.html#team-labels).

**Comment by aiuto** [comment]
> I believe current policy is to route Apple PRs touching core Bazel to
team-rules-CPP and assign to its team lead
<https://bazel.build/maintaining/maintainers-guide.html#team-labels>.
What I always try to do for reviews is use the blame layer to find out who
touched the code last and get them on the list. So sometimes that is the
Apple dev team and sometimes rules-cpp. So the bazel team member who
approves the PR should also approve the  CL.

The point of the ownership change is that apple related changes get
automatically seen by people who might understand them better than the
average bazel team member.  If Keith approves/rejects a change from an
unknown contributor, that is usually good enough for me to believe the
impact is desired and correct. There still may be other code quality
concerns. We need the bazel team member reviewer for those.

**Pull request closed without merging**